examples/many-updates/1-cafe-routes.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: gateway.networking.k8s.io/v1beta1
```

Comment:
@pleshakov This whole example directory is just for testing purposes. Feel free to skip. Or if you have a better script that creates a bunch of updates, please let me know!